scheduler-interface-spec.md

```diff
@@ -640,6 +641,7 @@ const (
 	ARCH                = "si/arch"
 	HostName            = "si/hostname"
 	RackName            = "si/rackname"
+       LabelName           = "si/labelname"
```

Comment:
this is the key name of the attribute, to be more self-explanatory, would it be better to rename this as `si/nodelabels`?